### ISSUE TITLE
Application in Medical images

### ISSUE THREAD

**Issue by javak87** [fenglinglwb/MAT#91]
Dear fenglinglwb,
First of all, thank you for your interesting package.
I would like to apply this package for medical images. In this domain, we have some images that are all of them cropped from the lower sides (20-40% of the lower part is missing). We want to complete this image using MAT. 
I applied the CelebA-HQ_512.pkl model and I got the following results:

![Screenshot from 2023-07-21 11-25-58](https://github.com/fenglinglwb/MAT/assets/44999772/69a8b7b8-a2b3-4ebc-9368-4cd61367bbe3)
Ground Truth

![Screenshot from 2023-07-21 11-26-30](https://github.com/fenglinglwb/MAT/assets/44999772/26fdae05-9638-4c00-b1df-953f5b681d53)
Applied Mask
![Screenshot from 2023-07-21 11-26-47](https://github.com/fenglinglwb/MAT/assets/44999772/ef1356b6-26d4-4cef-b733-f502e937edd5)
Result

Since in your training set, there were no medical images, I should train the network again. Therefore, I have two questions:
1- Do you think this method for our problem will be successful?
2- If it is possibly successful, training from scratch would be beneficial, or finetuning your existing pre-trained model?
Thank you for your time.
Best regards,
Javad

**Comment by yuji-byun** [comment]
Hello. Which approach did you choose? Did you train it from scratch or did you fine-tune it?
